pkg/client/connector/intercept.go

```diff
@@ -127,6 +129,41 @@ func (tm *trafficManager) workerPortForwardIntercepts(ctx context.Context) error
 	return nil
 }
 
+// startForwards starts port forwards and mounts for the given forwardKey. It returns true if either was started (port-forward or a mount), false otherwise
+func (tm *trafficManager) startForwards(ctx context.Context, wg *sync.WaitGroup, fk forwardKey, sftpPort int32, extraPorts []int32) bool {
+	started := false
+	if sftpPort > 0 {
+		// There's nothing to mount if the SftpPort is zero
+		started = true
+		mntCtx := dgroup.WithGoroutineName(ctx, fmt.Sprintf("/%s:%d", fk.PodIP, sftpPort))
+		wg.Add(1)
+		go tm.workerMountForwardIntercept(mntCtx, mountForward{fk, sftpPort}, wg)
+	}
+	for _, port := range extraPorts {
+		started = true
+		pfCtx := dgroup.WithGoroutineName(ctx, fmt.Sprintf("/%s:%d", fk.PodIP, port))
+		wg.Add(1)
+		go tm.portForward(pfCtx, wg, fk, port)
```

Comment:
The two names `workerMountForwardIntercept()` and `portForward()` aren't great as siblings here.